src/main/java/CollectionLesson.java

```diff
@@ -0,0 +1,67 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CollectionLesson {
```

Comment:
For separate tasks, we should create separate branches. This is class not related to HW6. It was in previous